apps/web/app/s/[surveyId]/layout.tsx

```diff
@@ -1,6 +1,24 @@
 import LegalFooter from "@/app/s/[surveyId]/components/LegalFooter";
+import { type Metadata } from "next/types";
 
-export default function SurveyLayout({ children }) {
+export const metadata: Metadata = {
+  title: "Formbricks",
```

Comment:
You have hard coded the meta data.

I think it makes more sense to create it dynamically using the Survey Title. Is this available on the public facing pages?

Reply:
Yup, it’s easily possible to change it to dynamic title. Will change it now